buildspec/csharpTests.yml

```diff
@@ -6,12 +6,12 @@ phases:
             dotnet: 3.1
     build:
         commands:
-            - dotnet build -c Release ./telemetry/csharp/ToolkitTelemetryGenerator.sln
-            - dotnet test -c Release ./telemetry/csharp/ToolkitTelemetryGenerator.sln --logger trx --results-directory ./.testresults
-
-reports:
-    DotnetTestExamples:
-        file-format: VisualStudioTrx
-        files:
-            - '**/*'
-        base-directory: './.testresults'
+            - echo "C# CodeBuild jobs removed until an image with .NET Framework and .NET Core can be created (using GitHub Actions in the meantime)"
```

Comment:
We do this in JB:

```
phases:
  install:
    runtime-versions:
      java: corretto11
      dotnet: 3.1

    commands:
      - apt update
      - apt install -y mono-complete
```

Reply:
Thank you, I didn't know about this. I'll explore it further outside of this PR.